src/Examine.Lucene/Suggest/Directories/SuggesterDirectoryFactoryBase.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Concurrent;
+using Lucene.Net.Store;
+
+namespace Examine.Lucene.Suggest.Directories
+{
+    public abstract class SuggesterDirectoryFactoryBase : ISuggesterDirectoryFactory
```

Comment:
This is public and should therefore have XML documentation - Use `/// <inheritdoc />` where appropriate

Reply:
Done